cmd/up.go

```diff
@@ -60,5 +72,10 @@ func (p *projectFactory) Create(c *cli.Context) (*project.Project, error) {
 	}
 
 	factory := &app.RancherProjectFactory{}
-	return factory.Create(c)
+	proj, err := factory.Create(c)
```

Comment:
if factory.Create(c) returns the same this as this method is returning, can't you just do
```
return factory.Create(c)
```